test/types/xarray.jl

```diff
@@ -0,0 +1,137 @@
+@testset "$ArrayType" for ArrayType in (AxisArray, KeyedArray)
+
+    x = ArrayType([1 2 3; 4 5 6], foo=["a", "b"], bar=[:x, :y, :z])
+
+    @test is_transformable(x)
+
+    @testset "apply" begin
+
+        @testset "OneToOne" begin
+            T = FakeOneToOneTransform()
+            @test FeatureTransforms.apply(x, T) == ones(2, 3)
```

Comment:
These transforms don't preserve type?

Reply:
it does in general, and reminder that 
```
julia> KA = KeyedArray(ones(3, 5), features=[:a, :b, :c], time=1:5);

julia> KA == ones(3, 5)
```

but as usual AxisArrays breaks the mould, and any operation on them doesn't return an AxisArray anyway
```julia
julia> A
2-dimensional AxisArray{Int64,2,...} with axes:
    :foo, ["a", "b"]
    :bar, [:x, :y, :z]
And data, a 2×3 Matrix{Int64}:
 1  2  3
 4  5  6

julia> A .^ 3
2×3 Matrix{Int64}:
  1    8   27
 64  125  216

julia> A .+ 1
2×3 Matrix{Int64}:
 2  3  4
 5  6  7
```

another reason to drop it